nusantara/nusa_datasets/karonese_sentiment/karonese_sentiment.py

```diff
@@ -0,0 +1,136 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from nusantara.nusa_datasets.karonese_sentiment.utils.karonese_sentiment_utils import \
+    map_label
+from nusantara.utils import schemas
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
+
+_CITATION = """\
+@article{karo2022sentiment,
+  title={Sentiment Analysis in Karonese Tweet using Machine Learning},
+  author={Karo, Ichwanul Muslim Karo and Fudzee, Mohd Farhan Md and Kasim, Shahreen and Ramli, Azizul Azhar},
+  journal={Indonesian Journal of Electrical Engineering and Informatics (IJEEI)},
+  volume={10},
+  number={1},
+  pages={219--231},
+  year={2022}
+}
+"""
+
+_DATASETNAME = "karonese_sentiment"
+
+_DESCRIPTION = """\
+Karonese sentiment was crawled from Twitter between 1 January 2021 and 31 October 2021.
+The first crawling process used several keywords related to the Karonese, such as
+"deleng sinabung, Sinabung mountain", "mejuah-juah, greeting welcome", "Gundaling",
+and so on. However, due to the insufficient number of tweets obtained using such
+keywords, a second crawling process was done based on several hashtags, such as
+#kalakkaro, # #antonyginting, and #lyodra.
+"""
+
+_HOMEPAGE = "http://section.iaesonline.com/index.php/IJEEI/article/view/3565"
+
+_LICENSE = "Unknown"
+
+_URLS = {
+    _DATASETNAME: "https://github.com/imkarokaro123/karonese/raw/main/karonese%20dataset.xlsx",
+}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+
+_SOURCE_VERSION = "1.0.0"
+
+_NUSANTARA_VERSION = "1.0.0"
+
+
+class KaroneseSentimentDataset(datasets.GeneratorBasedBuilder):
+    """Karonese sentiment was crawled from Twitter between 1 January 2021 and 31 October 2021.
+    The dataset consists of 397 negative, 342 neutral, and 260 positive tweets.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    NUSANTARA_VERSION = datasets.Version(_NUSANTARA_VERSION)
+
+    BUILDER_CONFIGS = [
+        NusantaraConfig(
+            name="karonese_sentiment_source",
+            version=datasets.Version(_SOURCE_VERSION),
```

Comment:
Since you had created the `SOURCE_VERSION` variable on line 56, you can use it in this line